src/libsyntax_pos/symbol.rs

```diff
@@ -514,18 +515,24 @@ fn with_interner<T, F: FnOnce(&mut Interner) -> T>(f: F) -> T {
 // by creating a new thread right after constructing the interner.
 #[derive(Clone, Copy, Hash, PartialOrd, Eq, Ord)]
```

Comment:
Argh, all the derives are now wrong...

Reply:
I just changed this back to `&'static str`.